vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/JsonResponseValidationSteps.java

```diff
@@ -224,21 +225,29 @@ public void saveElementsNumberByJsonPath(String jsonPath, Set<VariableScope> sco
     @When("I wait for presence of the element by JSON path '$jsonPath' in HTTP GET response from '$resourceUrl'"
             + " for '$duration' duration")
     public void waitForJsonFieldAppearance(String jsonPath, String resourceUrl, Duration duration) throws IOException
+    {
+        waitForJsonPath(jsonPath, resourceUrl, duration);
+        doesJsonPathElementsMatchRule(jsonPath, ComparisonRule.GREATER_THAN, 0);
+    }
+
+    private void waitForJsonPath(String jsonPath, String resourceUrl, Duration duration) throws IOException
```

Comment:
there is no much profit to extract this method now

Reply:
Just a bit more readable?